src/client/index.ts

```diff
@@ -646,6 +648,7 @@ export class Client {
 		// 	return arg;
 		// }
 		this.eventSender('tokens', tokens)
+		this.eventSender('tokens-loaded', Object.keys(tokens).length)
```

Comment:
@snowwhitedev This will always return the number of issuers configured in TN rather than the number of collections that have tokens, see TokenStore.getCurrentTokens method. Notice it returns all issuers with an empty array for unloaded issuers, or issuers that have 0 tokens. 

Have a look at TokenStore.getTotalTokenCount in PR 604, maybe you can create a new method TokenStore.getIssuersWithTokensCount which does a similar thing.

Reply:
@micwallace  I guess we have misunderstanding about the meaning of returning data. My expectation is to return the number of collections which are loaded regardless whether user has any tokens in that collection or not.
Let's say easily, there's only Perion collection in issuers config, Whenever I have any perion tokens or not, once Perion collection is loaded, `tokens-loaded` event returns 1. This is my expectation.
cc: @nicktaras